api/tests/unit/application/campaign/campaign-controller_test.js

```diff
@@ -127,23 +127,24 @@ describe('Unit | Application | Controller | Campaign', function () {
   describe('#getCsvAssessmentResults', function () {
     const userId = 1;
     const campaignId = 2;
-    const request = {
-      query: {
-        accessToken: 'token',
-      },
-      params: {
-        id: campaignId,
-      },
-      i18n: {
-        // TODO: Fix this the next time the file is edited.
-        // eslint-disable-next-line mocha/no-setup-in-describe
-        __: sinon.stub(),
-      },
-    };
+
+    let request;
 
     beforeEach(function () {
+      request = {
+        query: {
+          accessToken: 'token',
+        },
+        params: {
+          id: campaignId,
+        },
+        i18n: {
+          __: sinon.stub(),
+        },
+      };
```

Comment:
**suggestion** (pas bloquante du tout)

Plutôt que de mettre ça dans un beforeEach, ce qui oblige à faire des allers-retours pour comprendre à quoi correspond `request` à tel ou tel moment d'execution, j'aime bien mettre ce genre de chose dans une fonction privée définie à la fin du fichier avec un nom explicite du type `generateRequestForCampaignId`.

Comme ça dans le `it`, on a juste à appeler `const request = generateRequestForCampaignId(campaignId)` et en lisant juste cette ligne on comprend ce que c'est que cette variable `request` (et si on a un doute, on peut aller voir facilement la définition de la fonction).

Ici ça permettrait en plus d'éviter de dupliquer le code dans deux `beforeEach`.